### DISCUSSION TITLE
Youtube comment generator problem (Jquery and javascript)

### DISCUSSION THREAD

**Discussion by Subucc** [community/community#21392] [Other Feature Feedback, Questions, & Ideas]
<p>Hi, i have problem, i’m trying to write and program which will be genereting a something like this from prompt:</p>
<p>t</p>
<p>te</p>
<p>tes</p>
<p>test</p>
<p>tes</p>
<p>te</p>
<p>t</p>
<p>And i’ve done this. It work in console, but now i want to display it on the page. I know a little about JQuery, but i don’t know how to do that. Thanks for helping…</p>
<pre><code>&lt;!DOCTYPE html&gt;
&lt;html&gt;
&lt;head&gt;
&lt;title&gt;Generator&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;h1 id="testing"&gt;Hello world!&lt;/h1&gt;
&lt;script src="https://code.jquery.com/jquery-2.1.0.js"&gt;&lt;/script&gt;
&lt;script&gt;

var string = prompt ("Wpisz zdanie do przekonwertowania");
var text = string;
for (i = 0; i &lt; text.length - 1; i++) {
	console.log(text.slice(0,i))
}

for (i = 0; i &lt; string.length; i++) {
	var text = text.substring(0, text.length-1);
	console.log(text);
}

&lt;/script&gt;
&lt;/body&gt;
&lt;/hmtl&gt;
</code></pre>

**Comment by fire-eggs** [comment, marked as answer]
<p>Your javascript needs to generate HTML content to appear in the web page. <a href="https://www.w3schools.com/js/js_htmldom_nodes.asp" rel="nofollow noopener">Here</a> is a possible related tutorial.</p>

**Comment by sempostma** [comment]
<p> I’ve changed your code a bit. I replaced console.log with a document.write and made some changes to the first loop.</p>
<pre><code>&lt;!DOCTYPE html&gt;
&lt;html&gt;
&lt;head&gt;
&lt;title&gt;Generator&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&lt;h1 id="testing"&gt;Hello world!&lt;/h1&gt;
&lt;script src="https://code.jquery.com/jquery-2.1.0.js"&gt;&lt;/script&gt;
&lt;script&gt;

var string = prompt ("Wpisz zdanie do przekonwertowania");
var text = string;
for (i = 0; i &lt; text.length + 1; i++) {
	document.write(text.substring(0, i) + '
')
}

for (i = 0; i &lt; string.length; i++) {
	var text = text.substring(0, text.length -1);
	document.write(text + '
');
}

&lt;/script&gt;
&lt;/body&gt;
&lt;/hmtl&gt;
</code></pre>
